python/lsst/ts/tunablelaser/enums.py

```diff
@@ -31,20 +31,3 @@ class Output(enum.StrEnum):
     """A calibration energy level where the energy level adjusts."""
     MAX = "MAX"
     """Maximum energy level for the laser."""
-
-
-class OpticalConfiguration(enum.StrEnum):
```

Comment:
I suggest you leave this alone for now and just add a note here to remove this enum and use the one from ts-xml once that is available. The way you are going about this is a bit awkward. 

If you want to automatically switch the ts-xml once it is available, you can re-export it from here adding the try/except you included in the other files here and leaving the `OpticalConfiguration` entry in the `__all__` list above.